test/tools/yulInterpreter/EVMInstructionInterpreter.cpp

```diff
@@ -339,12 +344,18 @@ u256 EVMInstructionInterpreter::eval(
 	case Instruction::REVERT:
 		accessMemory(arg[0], arg[1]);
 		logTrace(_instruction, arg);
+		m_state.storage.clear();
```

Comment:
How "correct" is this interpreter? Shouldn't it only clear the changes to storage? Should we extract this change into its own PR?